language/move-vm/runtime/src/data_cache.rs

```diff
@@ -135,6 +135,10 @@ impl<'r, 'l, R: RemoteCache> TransactionDataCache<'r, 'l, R> {
         })
     }
 
+    pub(crate) fn mutated_accounts(&self) -> Vec<AccountAddress> {
```

Comment:
not sure about this cloning of keys just to find the count but I guess that is fine....

Reply:
Good point. I can just as well take the count of these keys which doesn't require cloning. I'll update.